lua/drop/config.lua

```diff
@@ -20,63 +81,349 @@ M.defaults = {
 
 M.ns = vim.api.nvim_create_namespace("drop")
 
+-- Convert current year to emoji equivalent for new_year theme.
+local function number_to_emoji(number)
+  local emojiMap = {
+    ["0"] = "0️⃣",
+    ["1"] = "1️⃣",
+    ["2"] = "2️⃣",
+    ["3"] = "3️⃣",
+    ["4"] = "4️⃣",
+    ["5"] = "5️⃣",
+    ["6"] = "6️⃣",
+    ["7"] = "7️⃣",
+    ["8"] = "8️⃣",
+    ["9"] = "9️⃣",
+  }
+  local result = ""
+  for i = 1, #number do
+    result = result .. emojiMap[number:sub(i, i)]
+  end
+  return result
+end
+local emojiYear = number_to_emoji(tostring(year))
```

Comment:
The new year theme includes the year as number emojis so we need to get the current year and map the numbers to the appropriate emoji.